### PR TITLE
twinkle.js: change default talkback heading to "New message"

### PR DESCRIPTION
**Pull request by DannyS712**
Solving #513. Does not affect user set preferences, just the default.

### CODE REVIEW

**Comment by Amorymeltzer** [comment]
@DannyS712 See discussion in #513

File: twinkle.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ Twinkle.defaultConfig.friendly = {
 	// Talkback
 	markTalkbackAsMinor: true,
 	insertTalkbackSignature: true,  // always sign talkback templates
-	talkbackHeading: "Talkback",
+	talkbackHeading: "New message",
```
Comment (Amorymeltzer) [inline twinkle.js]:
```suggestion
	talkbackHeading: "New message from " + mw.config.get('wgUserName'),
```
See discussion in #513

Comment (DannyS712) [reply twinkle.js]: I don't know how to accept change, but I agree with this

Comment (Amorymeltzer) [reply twinkle.js]: @DannyS712 	Is there not an "commit suggestion" button or something similar here in this window?

Comment (DannyS712) [reply twinkle.js]: Nope. There is "review changes" (which I think is for the overall pull request), in the top right, and when I clicked on the "start review" within this window it just adds a "1" to that button and still won't let me approve anything...

Comment (DannyS712) [reply twinkle.js]: Maybe because I am not a collaborator?

Comment (Amorymeltzer) [reply twinkle.js]: Should be the other way around, if anything — you own this branch.

Comment (Amorymeltzer) [reply twinkle.js]: Hmm, did you do something weird with this branch?  I notice above it says "unknown repository" for the location, which might be causing issues.

Comment (DannyS712) [reply twinkle.js]: I have no idea, sorry. Should I just open a new pull request incorporating your suggested change? Or, should this approved as is, and then another pull request with just your change can be committed? Sorry

Comment (DannyS712) [reply twinkle.js]: Okay. So should this be closed as rejected?

**Comment by Amorymeltzer** [comment]
Closed in #558

File: twinkle.js (hunk 1 above)
Comment (Amorymeltzer) [reply twinkle.js]: Thanks again for this

**Pull request closed without merging**